packages/sdk/src/chains/BaseAssetTransfer.ts

```diff
@@ -5,12 +5,25 @@ export abstract class BaseAssetTransfer {
   public config!: Config;
   public environment!: Environment;
 
+  /**
+   * Creates a Transfer<Fungible> object which can then be used to estimate transfer
+   * fees as well as create the necessary desposit and approval transactions.
+   *
+   * @category Bridge deposit
+   * @param {string} sourceAddress - The address of the sender
+   * @param {number} destinationChainId - The (Para)ChainId of the destination chain
+   * @param {string} destinationAddress - The address of the recipient on the destination chain
+   * @param {string} resourceId - The ID of the resource being transferred
+   * @param {string} amount - The amount of tokens to be transferred - This should already be scaled according to the Source chain decimals
```

Comment:
Kinda weird way to word it. 
Amount should be in the lowest denomination (wei, ...).
Something in that nature seems it would be more clear.

Reply:
Or amount should be in the denomination of the token.
Something to make it clear if the token has 12 decimals that gwei is used and not something else.

Reply:
If the Source chain token is configured with 12 decimals, the smallest possible transfer is `1 000 000 Wei` (1 MWei) (18-12=6).

If a user wants to transfer 1 ETH from that source chain, they would need to input `1_000_000_000_000`